Appraisals

```diff
@@ -6,7 +6,7 @@ appraise 'rails-3.2' do
   gem 'sqlite3', '~> 1.3.11', platforms: [:mri, :rbx]
 
   gem 'resque', '~> 1.25.2'
-  gem 'resque_spec', git: 'git@github.com:kyrylo/resque_spec.git'
+  gem 'resque_spec', github: 'kyrylo/resque_spec'
```

Comment:
FYI, I updated these because they are currently in SSH format, so `bundler` is trying to SSH-authenticate prior to fetching this git repo.